### PR TITLE
Added a missing Dense layer in the model.

### PR DESCRIPTION
**Pull request by masa-ita**
There should be a Dense layer with 16 units in the model, according to the description of layers.

### CODE REVIEW

File: site/en/tutorials/text/word_embeddings.ipynb [hunk 1]
```diff
@@ -597,7 +625,9 @@
         "else:\n",
         "  files.download('vecs.tsv')\n",
```
Comment (lamberta) [inline site/en/tutorials/text/word_embeddings.ipynb]: @MarkDaoust 
I'm getting an error here running in public colab: `MessageError: TypeError: Failed to fetch`

**Comment by masa-ita** [comment]
@lamberta @MarkDaoust 
I thought that changing file paths to absolute ones could solve this issue, but it fails again.
It seems that it will take some time before the files are available.
Relative paths worked second time.

**Comment by lamberta** [comment]
Thanks @masa-ita for looking into it. Can you save the notebook using Colab? It makes the diff easier to see. Either download or use the "Save in GitHub" feature.

**Comment by masa-ita** [comment]
@lamberta 
I revert the tsv file paths to relative and save from Colab.

File: site/en/tutorials/text/word_embeddings.ipynb (hunk 1 above)
Comment (MarkDaoust) [reply site/en/tutorials/text/word_embeddings.ipynb]: That's a Colab problem.

Sometimes it works the second time.

**Pull request merged**